charts/chainlink/templates/pod-monitor.yaml

```diff
@@ -10,4 +10,19 @@ spec:
       release: {{ .Release.Name }}
   podMetricsEndpoints:
     - port: access
-{{- end }}
\ No newline at end of file
+---
+{{- if .Values.db.enablePrometheusPostgresExporter }}
```

Comment:
does it work with this or `$` is needed?

Reply:
Yes, it will work fine, as it's not a nested block.